docs/release_notes/prophecy-libs.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Prophecy Libraries
+id: prophecy-libraries
+description: Prophecy library README
+sidebar_position: 4
+tags: [functionality, compatibility, version, library, plib, plibs, license]
+---
+
+Prophecy Libs are small and lightweight utility libraries, offering comprehensive sets of Spark functions tailored for Prophecy customers. They address the most common data transformation use cases, including monitoring, auditing, connectors, and other data processing functions. These versatile libraries support Scala and Python Spark environments.
+
+Here are the released artifacts:
+
+- Prophecy Libs for Scala: [Maven](https://mvnrepository.com/artifact/io.prophecy/prophecy-libs)
+- Prophecy Libs for Python: [PyPi](https://pypi.org/project/prophecy-libs/)
+
+For a list of the latest versions, see [Version Chart](/docs/release_notes/version_chart.md).
```

Comment:
How can the user know the libs version installed on the cluster?
(Aside from waiting until the cluster restarts?)

Reply:
Suggest: "IDentify which plibs version is running on your Spark cluster by checking the cluster dropdown in the pipeline."

Reply:
Done.